modules/preview-react/table/lib/Table.tsx

```diff
@@ -9,6 +8,22 @@ import {TableHeader} from './TableHeader';
 import {TableCell} from './TableCell';
 import {TableFooter} from './TableFooter';
 import {TableCaption} from './TableCaption';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+const tableStyles = createStyles({
+  display: 'grid',
+  border: `1px solid ${base.soap500}`,
+  borderRadius: system.shape.x2,
+  overflow: 'auto',
+  fontFamily: system.fontFamily.default,
+  fontSize: system.fontSize.subtext.large,
+  lineHeight: system.lineHeight.subtext.large,
+  fontWeight: system.fontWeight.normal,
```

Comment:
Why dont you just spreading `type.levels.subtext.large`?

Reply:
Yup!